providers/gce/gce_annotations.go

```diff
@@ -74,6 +74,13 @@ const (
 
 	// NetworkTierAnnotationPremium is an annotation to indicate the Service is on the Premium network tier
 	NetworkTierAnnotationPremium = cloud.NetworkTierPremium
+
+	// RBSAnnotationKey is annotated on a Service object to indicate
+	// opt-in mode for RBS NetLB
+	RBSAnnotationKey = "cloud.google.com/l4-rbs"
```

Comment:
Have we stabilized on this annotation? It's a bit obscure. Also, I think the standard prefix is "gke.io"

Maybe `gke.io/enable-l4-netlb-with-rbs`

Reply:
this has been merged to k/k in https://github.com/kubernetes/kubernetes/pull/106683 and cherry picked to patch releases

also this name was proposed in the RBS design doc.